src/platforms/salesforce/schemas.ts

```diff
@@ -524,6 +531,7 @@ export const salesforceEventRelationalSelect = {
   Opportunity: 'What.Id',
   Contact: 'Who.Id',
   Lead: 'Who.Id',
+  EventRelation: '(SELECT Id, RelationId FROM EventRelations)',
```

Comment:
Do we have e2e test for this?

Reply:
We have e2e on core, that's how I was able to catch these errors